packages/node/src/integrations/http/index.ts

```diff
@@ -30,6 +30,16 @@ interface HttpOptions {
    */
   spans?: boolean;
 
+  /**
+   * Whether the integration should create [Sessions](https://docs.sentry.io/product/releases/health/#sessions) for incoming requests to track the health and crash-free rate of your releases in Sentry.
+   * Read more about Release Health: https://docs.sentry.io/product/releases/health/
+   *
+   * Defaults to `true`.
+   *
+   * Note: If `autoSessionTracking` is set to `false` in `Sentry.init()` or the Client owning this integration, this option will also default to `false`.
```

Comment:
```suggestion
   * Note: If `autoSessionTracking` is set to `false` in `Sentry.init()` or the Client owning this integration, this option will be ignored.
```

or something like this? as in that case you can't overwrite this, right?